internal/task/package_task_factory.go

```diff
@@ -277,17 +277,17 @@ func applyLicenseContentRules(p *pkg.Package, cfg cataloging.LicenseConfig) {
 	for i := range licenses {
 		l := &licenses[i]
 		switch cfg.IncludeContent {
-		case cataloging.LicenseContentIncludeUnknown:
-			// we don't have an SPDX expression, which means we didn't find an SPDX license
-			// include the unknown licenses content in the final SBOM
-			if l.SPDXExpression != "" {
-				licenses[i].Contents = ""
-			}
 		case cataloging.LicenseContentExcludeAll:
 			// clear it all out
 			licenses[i].Contents = ""
```

Comment:
Should we flag licenses that have contents but no SPDX expression as `UNKNOWN`/`NOASSERTION`/`cataloging.UnknownStubValue`/ or something here? (Maybe that's already happening and I just didn't find it)

Reply:
@kzantow there are two places where this question applies: syft json and SPDX formats. For syft json there is no precedent for labeling licenses as "unknown", so I think that doesn't apply here. In terms of SPDX, this only applies for Other Licenses, which we area already claiming NOASSERTION when there are no contents (since we always include a LicenseRef- identifier).

Reply:
This was really more of a question than a request for a change, I'm ok with whatever behavior everyone else wants here, and agree maybe this is a SPDX concern, specifically.